src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double sideA;
+    private double sideB;
+    private double height;
+
+    public IsoscelesTrapezoid(double sideA, double sideB, double height) {
+        this.sideA = sideA;
+        this.sideB = sideB;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return (sideA + sideB) / 2 * height;
+    }
+
+    @Override
+    public String printInformation() {
+        return "Figure: RightTriangle, area: " + getArea()

```

Comment:
The `printInformation` method incorrectly refers to the figure as a 'RightTriangle'. It should be 'IsoscelesTrapezoid' to accurately describe the figure type.